about/policy/sgid/index.md

```diff
@@ -10,16 +10,33 @@ categories: []
 date: 2019-12-04 10:01:27 -0500
 ---
 
-The following applies to data in the internal SGID database.
+The following policies apply to data in the internal SGID geodatabase, which is a curated database of state-wide datasets. It is maintained as an internal, AGRC-only Esri Enterprise Geodatabase and is available to everyone as the cloud-hosted, PostGIS-backed [OpenSGID]({% link sgid/open-sgid/index.md %}). The contents of the SGID are also uploaded to the Utah SGID groups of our [ArcGIS Online Organization](https://utah.maps.arcgis.com).
 
-## Adding New Tables
+## Adding New Datasets to the SGID
 {: .text-left }
 
-When adding new data to the SGID, the steps outlined in the [Porter](https://github.com/agrc/porter) project will be followed.
+We strive to achieve and maintain a high standard of quality for the SGID. We use the following policies and procedures to guide this effort.
 
-Additional, related info:
+### Qualifying Datasets
+{: .text-left }
+
+Datasets should meet all the following criteria:
+
+- The dataset should cover a **state-wide** geographic extent, be part of a state-wide project or initiative (such as roads, address points, and parcels), or be relevant to broader projects or mandates that have a state-wide interest (e.g. geologic hazard mapping or endangered species mapping).
+- The dataset should be useful to **multiple agencies or organizations'** daily activities or decision-making. Datasets primarily used internally by a single agency aren't great candidates for the SGID.
+- The entity providing the dataset should be the **authoritative** or original source (for example, a county's parcels must come from the County government).
+- The entity providing the dataset does not have the **ability** to publicly share data (such as an existing ArcGIS Online organization). An entity's data storage costs or server capacity may not be sufficient justification for adding data to the SGID.
+
+You can still share datasets that don't meet the guidelines above with our [Open Data site](https://opendata.gis.utah.gov/). This is a great way to share data mainly intended for the general public with a wide audience. Please see our [Open Data sharing guidelines]({%link gis-data-sharing/open-data/index.md %}) for more information.
```

Comment:
```suggestion
Datasets can still be shared that don't meet the guidelines above with our [Open Data site](https://opendata.gis.utah.gov/). This is a great way to share data mainly intended for the general public with a wide audience. Please see our [Open Data sharing guidelines]({%link gis-data-sharing/open-data/index.md %}) for more information.
```

Reply:
I think this suggestion veers us back into passive voice. I think staying in active voice keeps the tone from being too officious or legalistic.

Reply:
OK, i was trying to replace the use of `you`

Reply:
Maybe "entities" or "agencies and organizations" instead?

Reply:
sure let's try that